search/src/Components/wrapper/upload.md

```diff
@@ -1,13 +1,20 @@
 ## Annotation Formats
```

Comment:
@geezacoleman would you mind checking the changes to this file? Thanks

Reply:
Only some minor changes suggested below